runtime/gc_base/StringTable.cpp

```diff
@@ -548,22 +548,27 @@ j9object_t
 j9gc_createJavaLangString(J9VMThread *vmThread, U_8 *data, UDATA length, UDATA stringFlags)
 {
 	J9JavaVM *vm = vmThread->javaVM;
+	J9InternalVMFunctions * const vmfcns = vm->internalVMFunctions;
```

Comment:
The variable name differs from (almost) every other - usually use vmFuncs.

Reply:
I think I copied it from somewhere but happy to change it to the usual convention

Reply:
Updated the variable name